app/kubemci/pkg/gcp/forwardingrule/forwardingrulesyncer.go

```diff
@@ -107,32 +107,22 @@ func (s *ForwardingRuleSyncer) ensureForwardingRule(lbName, ipAddress, targetPro
 func (s *ForwardingRuleSyncer) DeleteForwardingRules() error {
 	var err error
 	name := s.namer.HttpForwardingRuleName()
-	fmt.Println("Deleting http forwarding rule", name)
-	httpErr := s.frp.DeleteGlobalForwardingRule(name)
-	if httpErr != nil {
-		if utils.IsHTTPErrorCode(httpErr, http.StatusNotFound) {
-			fmt.Println("Http forwarding rule", name, "does not exist. Nothing to delete")
```

Comment:
I might be convinced otherwise, but the existing code of printing "does not exist" seems better than printing "deleted successfully".
Agreed that we should return a nil error in either case.

Reply:
I agree that "successfully deleted" isn't great if it didn't exist. My logic was that we should use a consistent pattern throughout. Do you want to have code like this for all the resources?

Reply:
Yes agreed that we should use the same pattern throughout.
Updating all resources sgtm.